docker-compose.yml

```diff
@@ -32,6 +32,7 @@ services:
     links:
       - database
       - mailhog
+      - skin_server_wmde19
```

Comment:
minor nitpick: since none of the rest is named `server` can we drop it from the title here as well `skin_wmde19`

Reply:
What about `skin_provider_wmde19` 😈  *\*runs away\**

Reply:
![triggered](https://media1.tenor.com/images/bac5a118662ca717b23636f8083d3bfe/tenor.gif?itemid=11478101)